buffer/src/main/java/io/netty/buffer/AdaptivePoolingAllocator.java

```diff
@@ -109,8 +109,8 @@ enum MagazineCaching {
      * This means the maximum amount of memory that we can have allocated-but-not-in-use is
      * 5 * {@link NettyRuntime#availableProcessors()} * {@link #MAX_CHUNK_SIZE} bytes.
      */
-    private static final int CENTRAL_QUEUE_CAPACITY = SystemPropertyUtil.getInt(
-            "io.netty.allocator.centralQueueCapacity", NettyRuntime.availableProcessors());
+    private static final int CENTRAL_QUEUE_CAPACITY = Math.min(2, SystemPropertyUtil.getInt(
```

Comment:
It should be `Math.max`

Reply:
@normanmaurer I'm afraid he's right. `Math.max` should have been used here

Reply:
🤦‍♂️ will fix

Reply:
Fixed ad104c6f8bc13a58c9f0d3695645efa61ce11dbe